src/Mod/TechDraw/Gui/ViewProviderDimension.cpp

```diff
@@ -121,17 +121,17 @@ void ViewProviderDimension::updateData(const App::Property* p)
 {
     if (p == &(getViewObject()->Type)) {
         if (getViewObject()->Type.isValue("DistanceX")) {
-            sPixmap = "TechDraw_Dimension_Horizontal";
+            sPixmap = "TechDraw_DistanceHorizontalDimension";

```

Comment:
Maybe "DistanceHorizontalDimension" is a bit long? Maybe it can just be "HorizontalDimension"? Adding the word "Distance" seems a bit redundant if we already add "Dimension".

Reply:
>  Adding the word "Distance" seems a bit redundant if we already add "Dimension".

Thanks for having a look. I improved the name now as you suggested.

> This seems like it's not using UTF8 codification as the ü in Jürgen is not displayed correctly.

This is now also fixed.